core/src/main/java/org/apache/brooklyn/core/mgmt/internal/LocalEntityManager.java

```diff
@@ -268,9 +270,26 @@ public void manage(Entity e) {
                     new Exception("source of duplicate management of "+e));
             return;
         }
+        // Validate all config before attempting to manage any entity. Do this here rather
+        // than in manageRecursive so that rebind is unaffected.
+        validateDescendantConfig(e);
```

Comment:
What's the reason to do it here rather than in `InternalEntityFactory` (analogously to the policy path)?